openc3/lib/openc3/models/script_status_model.rb

```diff
@@ -208,9 +208,9 @@ def update(force: false, queued: false)
     def destroy
       @destroyed = true
       undeploy()
-      self.class.store.hdel(@primary_key, @name)
+      self.class.store_queued.hdel(@primary_key, @name)
```

Comment:
Should take a queued argument like create and use the right one accordingly.

Reply:
There's only right one: queued. Otherwise you might get out of order when doing the delete.

Reply:
In the current code yes.   In the future who knows. It should be an argument just like for create. and pass true in the current use.